base/inc/AdePT/Utils.h

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * @file Utils.h
+ * @brief General utility functions.
+ * @author Andrei Gheata (andrei.gheata@cern.ch)
+ */
+
+#ifndef ADEPT_UTILS_H_
+#define ADEPT_UTILS_H_
+
+#ifndef VECCORE_CUDA_DEVICE_COMPILATION
+#include <atomic>
+#endif
+
+namespace adept {
+namespace utils {
+/**
+ * @brief Rounds up a value to upper aligned version
+ * @param value Value to round-up
+ */
+template <typename Type>
+VECCORE_ATT_HOST_DEVICE static Type round_up_align(Type value, size_t padding = 32)
```

Comment:
Why the arbitrary default value of 32 for padding?

Reply:
No reason, better no default